portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -11,22 +11,48 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 package com.google.sps.servlets;
-
+ 
 import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+ 
 /** Servlet that returns a greeting with my name. TODO(aabundis): modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+ 
+  private List<String> messages;
+ 
+  @Override
+  public void init() {
+    //Create new ArrayList and add messages.
+    messages = new ArrayList<>();
+    messages.add("The world is mine for the taking, make me king. -Eminem");
+    messages.add("Success is my only option, failure is not. -Eminem");
+    messages.add("I made it out of that world and found a place in this world. -Eminem");
+  }
+ 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Andres!");
+    String json = convertToJsonUsingGson();
+    // Send the JSON as the response
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+ 
+  /**
+   * Converts an ArrayList of messages into a JSON string using the Gson library.
+   */
+  private String convertToJsonUsingGson() {
+    Gson gson = new Gson();
+    String json = gson.toJson(messages);
+    return json;
```

Comment:
I think you could consolidate this to just return the result, rather than initializing another string variable.

```suggestion
    return gson.toJson(messages);
```

Reply:
Ack!